BackofficeBundle/Form/Type/AbstractOrchestraGroupType.php

```diff
@@ -1,19 +0,0 @@
-<?php
```

Comment:
You should not have suppressed this class !! it was mandatory for the group choice form type